docs/tutorials/prowler-app-mute-findings.md

```diff
@@ -56,4 +56,358 @@ If the YAML configuration is invalid, an error message will be displayed
 ![Check muted fidings](../img/mutelist-ui-9.png)
 
 ???+ note
-    The Mutelist configuration takes effect on the next scans.
\ No newline at end of file
+    The Mutelist configuration takes effect on the next scans.
+
+## Mutelist Config File
+
+The Mutelist configuration uses YAML format to define which findings should be muted. Below are examples for different cloud providers supported by Prowler App.
+
+### AWS Provider Examples
```

Comment:
You can then remove examples from each heading if the above one is called like that.

Reply:
Done